### PR TITLE
Buffs strangling

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
tweak: Strangling has been buffed considerably. It now does 4 oxygen damage per tick instead of 3, can no longer be resisted out of (as it was prebase) and you can no longer call out for help over radio due to having a crushed windpipe (but you can still scream, for some reason).
/:cl:

Strangling is now a viable way to kill people again! Sometimes it's more fun to squeeze the life out of someone than just beating them to death.

### CODE REVIEW

File: hippiestation/code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/mob/living/carbon/breathe()
+	if(!getorganslot("breathing_tube"))
```
Comment (KayeArray) [inline hippiestation/code/modules/mob/living/carbon/life.dm:2]: this is actually a thing?

Comment (PopNotes) [reply hippiestation/code/modules/mob/living/carbon/life.dm]: it's an implant, yeah

File: hippiestation/code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/mob/living/can_speak_vocal(message)
+	if(disabilities & MUTE)
+		return 0
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/say.dm]: FALSE

File: hippiestation/code/modules/mob/living/say.dm [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+	if(is_muzzled())
+		return 0
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/say.dm]: FALSE

File: hippiestation/code/modules/mob/living/say.dm [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+	if(!IsVocal())
+		return 0
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/say.dm]: FALSE

File: hippiestation/code/modules/mob/living/say.dm [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+	if(pulledby && pulledby.grab_state == GRAB_KILL)
+		return 0
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/say.dm]: FALSE

File: hippiestation/code/modules/mob/living/say.dm [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+		
+	return 1
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/say.dm]: TRUE

File: hippiestation/code/modules/mob/living/living.dm [hunk 7]
```diff
@@ -1,3 +1,15 @@
...
\ No newline at end of file
+	return
+	
+/mob/living/resist_grab(moving_resist)
+	if(pulledby.grab_state)
+		if(prob(30) && pulledby.grab_state != GRAB_KILL)
+			visible_message("<span class='danger'>[src] has broken free of [pulledby]'s grip!</span>")
+			add_logs(pulledby, src, "broke grab")
+			pulledby.stop_pulling()
+			return 0
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/living.dm]: FALSE

**Comment by zamolxius** [comment]
superb,  it really grinds my gears when I hold somebody by the neck and they just "resist"  and run away

**Comment by McBawbaggings** [comment]
sure
+1

**Comment by Dudeinyourbackyard** [comment]
hell yeah +1

**Comment by JamieH** [comment]
Tested?

**Pull request merged**